socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -112,7 +122,13 @@ export default Vue.component("author-bar", {
             }
             this.$http.post(`/api/profiles/${this.currentBrowsingProfileId}/add_follower/`, {guid: this.author.guid})
                 .then(() => this.isUserFollowingAuthor = true)
-                .catch(err => console.error(err) /* TODO: Proper error handling */)
+                .catch(error => {
+                    const msg = "An error happened while trying to follow ${user}"
+                    const translated = gettext(msg)
+                    Vue.snotify.error(_template(translated)({user: this.authorNameOrHandle}))
+                    console.error(`${_template(msg)({user: this.authorNameOrHandle})}; error met: ${error}`)
```

Comment:
I think console logging is only useful for development or if we plan to do some better notification but don't have it yet. Now that this does, I think the console logger should be removed.

Reply:
I'm not in favor of displaying the stacktrace to the user. But Logging it in console could help us in case of a nasty bug. Dunno. You decide.

Reply:
Definitely no stacktrace to users - it wont be helpful to anyone. But I'm not sure outputting a stacktrace to the console will help either. Users wont know to look there.

Having `console` calls can even crash the JS code on certain browsers, like [IE9 and below](https://caniuse.com/#feat=console-basic). Of course, we don't really care about such old browsers since our code likely will crash on them anyway, but still, good to keep in mind. Maybe this was the reason in Diaspora.

I'm in favour of generally not logging stuff to the console. But then we're in highly development mode. Maybe in some cases it makes sense to be able to see production errors until code is seen as more stable. I don't think this is such a place since the error is likely just going to be a backend error message or a network error. The former can be seen by the node admin. The latter can be probably seen from the network call panel, if needed. So, for this one, I would say remove it 👍

Reply:
Duly noted. Removing `console` directives, then (and hopping nobody uses our beautiful application on that IE crap anyway).